src/inverseRobot.js

```diff
@@ -7,7 +7,17 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const invertedRobot = {};
+
+  for (const key in robot) {
+    if (robot[key] in invertedRobot) {
```

Comment:
It's a good practice to use 'hasOwnProperty' method to check for property existence to avoid issues with properties from the prototype chain. Consider using 'invertedRobot.hasOwnProperty(robot[key])' instead of 'robot[key] in invertedRobot'.